### DISCUSSION TITLE
开放平台为什么要感知用户 namespace.lock.swich限制开放平台修改发布

### DISCUSSION THREAD

**Discussion by PepoRobert** [apolloconfig/apollo#4076] [Q&A] [answered]
- [x] 我已经检查过[discussions](https://github.com/ctripcorp/apollo/discussions)
- [x] 我已经搜索过[issues](https://github.com/ctripcorp/apollo/issues)
- [x] 我已经仔细检查过[FAQ](https://www.apolloconfig.com/#/zh/faq/common-issues-in-deployment-and-development-phase)

**描述bug**

简洁明了地描述一下bug

**复现**

通过如下步骤可以复现：

1. namespace.lock.swich=true
2. 通过开放平台修改后发布
3. 
4. 

**期望**

开放平台之前修改人和发布人都是系统固定的同一个人。开放平台为什么要感知用户 

**截图**

如果可以，附上截图来描述你的问题

### 额外的细节和日志

- 版本：
- 错误日志
- 配置：
- 平台和操作系统

**Comment by nobodyiam** [comment, marked as answer]
This is as designed. Open platform's token represents the third party system and the modified by should represent the actual operator.

**Comment by PepoRobert** [comment]
好的 谢谢
